framework/wazuh/exception.py

```diff
@@ -207,9 +207,9 @@ class WazuhException(Exception):
                'remediation': 'Please, use `GET /agents?select=id,name` to find all available agents'
                },
         1702: {'message': 'Unable to restart agent(s)',
-               'remediation': 'Please make sure the agent exists. it is active and it is not the manager(Agent 000)'
+               'remediation': 'Please make sure the agent exists, it is active and it is not the manager (agent 000)'
```

Comment:
```suggestion
               'remediation': 'Please make sure the agent exists, it is active and it is not the master node (agent 000)'
```

Reply:
This error comes out when trying to restart agents. If the user does not have the cluster deployed, it could be confusing to mention "master node" instead of manager.